native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNav.js

```diff
@@ -121,10 +130,12 @@ class BpkHorizontalNav extends React.Component {
         return null;
       }
       return (
-        <AnimatedIndicator
-          xOffset={this.state.indicatorOffsetX}
-          width={this.state.indicatorWidth}
-        />
+        <View style={styles.indicatorWrapper}>
```

Comment:
What if we applied the style to here instead?
https://github.com/Skyscanner/backpack/blob/a2ec249ab31ad13abed7f16a70837b8d6140af4c/native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNav.js#L150

Then would we not need the additional view?

Reply:
I'll try that 👍